classes/forbidden_shop/ForbiddenShopAPIPresenter.php

```diff
@@ -26,6 +27,19 @@ public static function eventDataResponse()
             ],
         ];
     }
+    /**
+     * @throws RuntimeException
+     */
+    public static function exchangeDataResponse()
+    {
+        return [
+            'exchangeData' => [
+                'ayakashiFavor' => ForbiddenShopManager::AYAKASHI_FAVOR,
+                'favorExchange' => ForbiddenShopManager::FAVOR_EXCHANGE,
+                'factionMissions' => ForbiddenShopManager::FACTION_MISSIONS
```

Comment:
Huh, a little confused why we need to return static variables in an API response. How will these be used in the frontend - Could we pass them in as props during component initialization?

Reply:
Good point, API isn't necessary here.

Reply:
Fixed in commit.